examples/conditional-request-handling/ConditionalRequestHandling_Client.cpp

```diff
@@ -25,22 +25,27 @@ int main(int argc, char* argv[]) {
   FLAGS_minloglevel = 0;
   folly::init(&argc, &argv);
 
+  folly::SocketAddress address;
+  address.setFromHostPort(FLAGS_host, FLAGS_port);
+
   auto rsf = RSocket::createClient(
-      TcpConnectionFactory::create(FLAGS_host, FLAGS_port));
+      std::make_unique<TcpConnectionFactory>(std::move(address)));
   auto rs = rsf->connect().get();
 
   LOG(INFO) << "------------------ Hello Bob!";
   auto s1 = yarpl::Reference<ExampleSubscriber>(new ExampleSubscriber(5, 6));
   rs->requestStream(Payload("Bob"))
       ->take(5)
-      ->subscribe(yarpl::Reference<yarpl::flowable::Subscriber<Payload>>(s1.get()));
+      ->subscribe(
+          yarpl::Reference<yarpl::flowable::Subscriber<Payload>>(s1.get()));
```

Comment:
(This was a clang-format change)

Unfortunately, Reference<Derived> can't be upcast to Reference<Base>, you'll just get a compiler error.  That's why most Subscriber code gets the raw Subscriber pointer and makes a new Reference out of it.  I have a diff in the works to fix that, just need to fix some outstanding test issues first.